NEWS.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## [7.5.0] - 2016-08-03
+
+### Added
+
+* Add support for the new `electron` package name. #435
```

Comment:
> Support for the new `electron` package name (#435)

I'm trying not to be redundant with the header name. (Doesn't always happen, sadly)